docs/DeveloperGuide.md

```diff
@@ -591,6 +589,22 @@ Use case ends.
   
     Use case ends.
 
+**Use case: Sort Students**
+
+**MSS**
+
+1. User requests to list students.
+2. T_Assistant shows a list of students.
+3. User requests to sort students.
+4. T_Assistant sorts students in ASCII order.
```

Comment:
Sorry I'm blocking this PR, can you update that the message stated in the UG is wrong too

Reply:
For both sort student and group in the UG, currently the screenshots show that the it is alphabetical order.

Can you add a note that states it is ASCII not alphabetical and that the ss is wrong

Reply:
Ok, should I just update the screenshot in that case?